digitalocean/database/resource_database_cluster.go

```diff
@@ -661,6 +675,16 @@ func buildMongoDBConnectionURI(conn *godo.DatabaseConnection, d *schema.Resource
 	return uri.String(), nil
 }
 
+// DO API returns null values for uri and private uri for Kafka clusters.
+// buildKafkaConnectionURI sets the uri and private uri using connection's host.
+func buildKafkaConnectionURI(conn *godo.DatabaseConnection, port int) string {
+	host := conn.Host
+
+	uri := fmt.Sprintf("%s:%d", host, port)
```

Comment:
Lets make sure that this is consistent with what the DBaaS team will eventually return from the API. Should there be a scheme? The other engines all seem to use the format `scheme://user:pass@host:port`

Reply:
> Lets make sure that this is consistent with what the DBaaS team will eventually return from the API. Should there be a scheme? The other engines all seem to use the format `scheme://user:pass@host:port`

Converting this PR to a draft as dbaas said they might actually get to it this week or next!

Reply:
Sounds good!